recipe/bld.bat

```diff
@@ -68,6 +68,18 @@ rem copy out32dll\libeay32.dll %LIBRARY_BIN%\libeay32.dll
 rem mkdir %LIBRARY_INC%\openssl
 rem xcopy /S inc32\openssl\*.* %LIBRARY_INC%\openssl\
 
```

Comment:
Would it be better to increase verbosity by adding `@echo on` at the top of the file as there are a lot of commands to run in bld.bat and not all of them are visible now in the logs